modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -576,28 +627,26 @@ function SubjectMemberInternal (props) {
                                              size="small"
                                            />
                                          })}
-                                       </React.Fragment> },
-                  { title: 'Actions',
-                    cellStyle: {
-                      padding: '0',
-                      whiteSpace: 'nowrap',
-                      textAlign: 'end'
-                    },
-                    render: rowData => <React.Fragment>
-                                         <EditButton
-                                           entryPath={rowData["@path"]}
-                                           entryType="Form"
-                                         />
-                                         <DeleteButton
-                                           entryPath={rowData["@path"]}
-                                           entryName={getEntityIdentifier(rowData)}
-                                           entryType="Form"
-                                           warning={rowData ? rowData["@referenced"] : false}
-                                           onComplete={fetchTableData}
-                                         />
-                                       </React.Fragment> },
+                                       </>) },
                 ]}
-               />
+                enableRowActions
+                positionActionsColumn="last"
+                renderRowActions={({ row }) => (
+                    <Box sx={{ display: 'flex', flexWrap: 'nowrap', gap: '8', float: 'right' }}>
```

Comment:
In `sx`, `gap` needs to be specified with a unit of measurement (e.g. `8px`), a number alone is invalid. So `gap: '8'` has no effect here. Which is fine because it looks good without a gap, so it can just be reoved.
```suggestion
                    <Box sx={{ display: 'flex', flexWrap: 'nowrap', float: 'right' }}>
```

The styling needs to be adjusted on the actions cell though, because the delete buttons are not only misaligned with the top menu, but also look cut off on hover:

![image](https://user-images.githubusercontent.com/651980/233133419-885562c2-d2ae-4e0a-b8a8-17345324bd86.png)

I believe we need a 16px right padding on the cell.